### ISSUE TITLE
Give Twitch API values precedence over cached values

### ISSUE THREAD

**Issue by Camtendo** [The-Velvet-Room/web-overlay#126]
There are instances where certain fields of polled Twitch data will not update because the cached values are already set. We need to default to always setting from values return from the API.

**Comment by dguenther** [comment]
I'm not sure what this is referring to. It looks like we don't poll for channel info though, so if we set the game or status from another source, I'm not sure it'll update.

I'm working on a refactor of the Twitch socket that breaks out the Twitch and Redis calls into a separate file, so hopefully this will get fixed in the process. I'm not sure it'll be ready for SoDak, though.

**Comment by Camtendo** [comment]
Yeah, that sounds like what the issue is. The current workaround is to clear Twitch stuff, then reset it, so we're fine for Sodak
